lms/djangoapps/instructor_task/api_helper.py

```diff
@@ -25,7 +25,26 @@
 
 class AlreadyRunningError(Exception):
     """Exception indicating that a background task is already running"""
-    pass
+
+    message = 'Requested task is already running'
```

Comment:
Please see [our developer guidelines on internationalization](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html).  

Essentially, for all strings that are presented to users, they need to be surrounded by `gettext()` or `_()` so they can be translated correctly. You'll want to do this for all messages in this file and all strings that are presented to users.